CourseSketchProjects/CourseSketchLoginServer/src/main/java/coursesketch/database/DatabaseClient.java

```diff
@@ -232,14 +232,13 @@ private static String getUserInfo(final DBObject cursor, final boolean loginAsDe
      *            The email of the user.
      * @param isInstructor
      *            If the default account is an instructor
-     * @throws GeneralSecurityException
-     *             Thrown if there are problems creating the hash for the
-     *             password.
+     * @throws AuthenticationException Thrown if an invalid key is set
+     * @throws NoSuchAlgorithmException Thrown if the specified algorithm does not exist.
      * @throws RegistrationException
      *             Thrown if the user already exist in the system.
      */
     public static final void createUser(final String user, final String password, final String email, final boolean isInstructor)
-            throws GeneralSecurityException, RegistrationException {
+            throws AuthenticationException, NoSuchAlgorithmException, RegistrationException {
```

Comment:
Why should I remove this exception?
On Mar 30, 2016 2:36 PM, "areftw" <notifications@github.com> wrote:

> In
> CourseSketchProjects/CourseSketchLoginServer/src/main/java/coursesketch/database/DatabaseClient.java
> <https://github.com/tracyhammond/coursesketch/pull/470#discussion_r57949686>
> :
>
> >       * @throws RegistrationException
> >       *             Thrown if the user already exist in the system.
> >       */
> >      public static final void createUser(final String user, final String password, final String email, final boolean isInstructor)
> > -            throws GeneralSecurityException, RegistrationException {
> > +            throws AuthenticationException, NoSuchAlgorithmException, RegistrationException {
>
> Removed the GeneralSecurityException?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/tracyhammond/coursesketch/pull/470/files/5cc9d3d3675910e4833fc310a0cc9e78bffaab80#r57949686>
>